tests/mocked/test_uptime.py

```diff
@@ -0,0 +1,224 @@
+import responses
+
+from pydo import Client
+
+
+@responses.activate
+def test_uptime_check_lists_get(mock_client: Client, mock_client_url):
```

Comment:
if you run `make lint-tests`, you'll see a bunch of errors asking:
`tests/mocked/test_uptime.py:42:0: C0116: Missing function or method docstring (missing-function-docstring)`
This is super easy fix, you'll just have to add "docstrings" under the function, as such:
```suggestion
def test_check_state_get(mock_client: Client, mock_client_url):
    """Mock Uptime Check State"""
```